__tests__/api.test.js

```diff
@@ -167,3 +167,46 @@ describe('GET/api/articles/:article_id/comments', () => {
         })
     });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+    it('200: responds with the updated article', () => {
+        const newVote = { inc_votes: 5 };
+
+        return request(app).patch('/api/articles/1').send(newVote)
+        
+        .then((response) => {
+        const {article} = response.body
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("body");
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("votes");
+            expect(article).toHaveProperty("article_img_url");
+        })
+    });
```

Comment:
When testing the properties we should also test the value of the properties we know, ie article_id and votes 
if you look at jest documentation you will see that toHaveProperty takes a second argument which will help you check the values